File: java-api-examples/VersionTest.java
// Copyright 2025 Xiaomi Corporation

import com.k2fsa.sherpa.onnx.*;

public class VersionTest {
  public static void main(String[] args) {
    System.out.printf("sherpa-onnx version: %s\n", VersionInfo.getVersion());
    System.out.printf("sherpa-onnx gitSha1: %s\n", VersionInfo.getGitSha1());
    System.out.printf("sherpa-onnx gitDate: %s\n", VersionInfo.getGitDate());
  }
}

File: python-api-examples/version-test.py
#!/usr/bin/env python3

# Copyright (c)  2026  Xiaomi Corporation

import sherpa_onnx


def main():
    print(f"sherpa-onnx version: {sherpa_onnx.version}")
    print(f"sherpa-onnx git sha1: {sherpa_onnx.git_sha1}")
    print(f"sherpa-onnx git date: {sherpa_onnx.git_date}")
    print(f"onnxruntime version: {sherpa_onnx.onnxruntime_version}")


if __name__ == "__main__":
    main()
